### ISSUE TITLE
Add switches

### ISSUE THREAD

**Issue by syssi** [syssi/esphome-jbd-bms#10]
Traffic capture of the charging/discharging switch: https://github.com/syssi/esphome-jbd-bms/blob/main/docs/pdus/JBD-SP04S034-L4S-200A-B-U_MOS.txt

**Comment by syssi** [comment]
```
>>> dd5a00025678ff3077
<<< dd000000000077
>>> dd5a2d020002ffcf77 # Balacing off
<<< dd2d0000000077
>>> dd5a01020000fffd77
<<< dd010000000077

>>> dd5a00025678ff3077
<<< dd000000000077
>>> dd5a2d020006ffcb77 # Balancing on
<<< dd2d0000000077
>>> dd5a01020000fffd77
<<< dd010000000077

>>> dd5a00025678ff3077
<<< dd000000000077
>>> dd5a2d020002ffcf77 # Balancing off
<<< dd2d0000000077
>>> dd5a01020000fffd77
<<< dd010000000077

>>> dd5a00025678ff3077
<<< dd000000000077
>>> dd5a2d020006ffcb77 # Balancing on
<<< dd2d0000000077
>>> dd5a01020000fffd77
<<< dd010000000077
```

**Comment by syssi** [comment]
Resolved by #36
